Correct some errno declarations for FreeBSD.

===================================================================
RCS file: /home/jdp/m3-cvs/m3/m3core/src/unix/freebsd-2/Uerror.i3,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 Uerror.i3
--- m3/m3core/src/unix/freebsd-2/Uerror.i3.orig	1996/09/24 05:22:00	1.1.1.1
+++ m3/m3core/src/unix/freebsd-2/Uerror.i3	1996/09/24 05:32:41
@@ -23,7 +23,7 @@
   ENOEXEC = 8;                 (* Exec format error *)
   EBADF   = 9;                 (* Bad file number *)
   ECHILD  = 10;                (* No children *)
-  EAGAIN  = 11;                (* No more processes *)
+  EDEADLK = 11;                (* Resource deadlock avoided *)
   ENOMEM  = 12;                (* Not enough core *)
   EACCES  = 13;                (* Permission denied *)
   EFAULT  = 14;                (* Bad address *)
@@ -51,7 +51,8 @@
   ERANGE  = 34;                (* Result too large *)
 
  (* non-blocking and interrupt i/o *)
-  EWOULDBLOCK  = 35;           (* Operation would block *)
+  EAGAIN       = 35;           (* Resource temporarily unavailable *)
+  EWOULDBLOCK  = EAGAIN;       (* Operation would block *)
   EINPROGRESS  = 36;           (* Operation now in progress *)
   EALREADY     = 37;           (* Operation already in progress *)
 
@@ -108,29 +109,22 @@
   EPROCUNAVAIL    = 76;          (* Bad procedure for program *)
 
   (* POSIX errnos *)
-  ENOLCK  = 77;                  (* LOCK_MAX exceeded     *)
-
-  (* IPC errors *)
-
-  (* I don't know about the following codes. ow 02.10.1994 *)
-  ENOMSG = 78;                   (* No message of desired type *)
-  EIDRM  = 79;                   (* Identifier removed *)
-
-  (* Alignment error of some type (i.e., cluster, page, block ...) *)
-  EALIGN  = 80;                  (* alignment error *)
-
-  (* System V mappings from BRL package *)
-  EDEADLK  = EWOULDBLOCK;        (* resource deadlock would occur *)
+  ENOLCK          = 77;          (* No locks available *)
+  ENOSYS          = 78;          (* Function not implemented *)
 
+  EFTYPE          = 79;          (* Inappropriate file type or format *)
+  EAUTH           = 80;          (* Authentication error *)
+  ENEEDAUTH       = 81;          (* Need authenticator *)
+  ELAST           = 81;          (* Must be equal to the largest errno *)
 
 <*EXTERNAL*>
 VAR
   errno: int;
 
 
-(* Extention by mjordan *)
+(* Extension by mjordan *)
 CONST
-  Max = ENOLCK; (* should be exported from Uerror *)
+  Max = ELAST; (* should be exported from Uerror *)
   
 <*EXTERNAL*> VAR
   sys_nerr: int;
